src/renderer/assets/styles/reader-app.css

```diff
@@ -285,6 +285,55 @@
   }
 }
 
+.menu_option {
+    line-height: 0.9rem;
+    padding: 0;
+    margin: 0;
+    float: right;
+
+    & > li {
+      display: inline-block;
+      height:100%;
+      width:60px;
+      font-weight: 600;
+      color:#c1c1c1;
+      border-left:1px solid #c1c1c1;
+      cursor: pointer;
+
+      & > button {
+        width: inherit;
+        height: 60px;
+        color:inherit;
+        background-color: transparent;
+        border: none;
+        cursor: pointer;
+
+        & svg {
+          width:30px;
+          padding:0;
+          margin: 0 auto;
+          height:30px;
+        }
+      }
+      & .active {
+        color:black;
+        border-bottom:3px solid black;
+        padding-bottom:15px;
+      }
+
+      & .right {
+        float:right;
+        border-left:1px solid #c1c1c1;
+        border-right:none;
+      }
+
+      & .blue {
+        background-color: #2B5366;
+      }
```

Comment:
The CSS in the codediff above looks like a copy-paste from `.main_navigation ul`:

https://github.com/readium/readium-desktop/blob/3bab0fd6d84ce1006e155a2885f1d93259586240/src/renderer/assets/styles/reader-app.css#L245-L284

Let's avoid duplicating CSS.
The only CSS rule that your proposed nested `ul.menu_option` CSS selector adds is: `float: right;`.
So, we can simply use the `,` CSS selector combinator, and apply the `float` as follows:

```
  & > ul, & ul.menu_option {
    &.menu_option {
      float: right;
    }
```

(line 240, inside `.main_navigation`, replace `& > ul {` with the above snippet of CSS code)